db/schema.rb

```diff
@@ -10,7 +10,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_05_24_011519) do
+ActiveRecord::Schema.define(version: 2019_06_30_161849) do
+
+  create_table "active_storage_attachments", force: :cascade do |t|
```

Comment:
What's the purpose of these? I don't see them added by any migrations so if these tables are required for the uploading to work we need to create the tables with migrations.

We will get runtime issue in docker if the database is changed through the `schema.rb`. The way it works is the the `schema.rb` can be used to setup the database from a clean state (faster then running through the migrations one-by-one). But once the database exists, all changes must be performed through migrations.

Reply:
Oh, good catch. That's from my tinkering with ActiveStorage, rails core file uploading solution. I guess I forgot to revert that after I decided ActiveStorage wasn't a good solution. I'll remove that.

Reply:
Good that it's an easy revert 😄